src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -40,17 +43,16 @@ export class HuiDialogEditCard extends LitElement {
   @property() private _params?: EditCardDialogParams;
 
   @property() private _cardConfig?: LovelaceCardConfig;
+  @property() private _view!: LovelaceViewConfig;
 
```

Comment:
`_viewConfig`

Reply:
Sure I will rename that attribute

Reply:
Fixed with f5d3aa0e830edd5130ecc2a5e5c534b9f135304a